docs/getting-started.md

```diff
@@ -60,3 +60,82 @@ Now, it's your turn!
 <figure><img src=".gitbook/assets/admin-dashboard.png" alt="Admin dashboard overview"></figure>
 
 </div>
+
+### Using AssetMapper
+
+You may need to disable the Stimulus App started on `sylius/bootstrap-admin-ui` package and start it yourself to prevent twice Ajax calls.
```

Comment:
```suggestion
To prevent duplicate Ajax calls, disable the auto-initialized Stimulus app and Symfony UX stylesheets from the `sylius/bootstrap-admin-ui` package, so you can take control of Stimulus initialization in your own code.
```